lib/appsignal/hooks/sidekiq.rb

```diff
@@ -14,23 +14,21 @@ class args retried_at failed_at
       end
 
       def call(_worker, item, _queue)
-        args =
-          if item["class"] == "ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper"
-            item["args"].first["arguments"]
-          else
-            item["args"]
-          end
-        params = Appsignal::Utils::ParamsSanitizer.sanitize args,
+        job = ::Sidekiq::Job.new(item)
+
+        params = Appsignal::Utils::ParamsSanitizer.sanitize(
+          job.display_args,
```

Comment:
Small heads up, I noticed that the original args were mutated by the sanitizer, causing `display_class` to fail later. Not sure if it was actually the sanitizer causing it. It was consistent so if you don't run into it with a real sidekiq job it's probably no longer a problem.